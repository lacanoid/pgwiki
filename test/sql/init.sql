CREATE EXTENSION wiki cascade;
\pset tuples_only

\dx wiki


