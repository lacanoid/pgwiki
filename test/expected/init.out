CREATE EXTENSION wiki cascade;
NOTICE:  installing required extension "plperlu"
\pset tuples_only
\dx wiki
 wiki | 0.1     | wiki   | SQL wiki

